app/models/assignment.rb

```diff
@@ -46,6 +46,7 @@ class Assignment < ActiveRecord::Base
   # is no parent (the same holds for the child peer reviews)
   belongs_to :parent_assignment, class_name: 'Assignment', inverse_of: :pr_assignment
   has_one :pr_assignment, class_name: 'Assignment', foreign_key: :parent_assignment_id, inverse_of: :parent_assignment
+  has_many :peer_reviews, through: :groupings
```

Comment:
This is interesting. Groupings do have to different named associations to peer reviews, right? (as_reviewer and as_reviewee, or something)

I'm not sure exactly what you want, but I think you'd want to specify a `:source` option for the `has_many` - http://guides.rubyonrails.org/association_basics.html#options-for-has-many-source